src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -1071,29 +1038,48 @@ namespace
 
         try
         {
-            if (extension == ".exr")
-            {
-                write_exr_image(
-                    bf_file_path,
-                    image,
-                    image_attributes);
+            bool high_dynamic_range_format = false;
+            if (extension == ".exr" || extension == ".tiff" || extension == ".tif" || extension == ".hdr")
+                high_dynamic_range_format = true;            
+
+            create_parent_directories(bf_file_path);
+
+            const std::string filename = bf_file_path.string();
+            GenericImageFileWriter writer(filename.c_str());
+
+            // Add image to writer
+            Image *transformed_image;
+            if (!high_dynamic_range_format && image.properties().m_channel_count == 4) {
```

Comment:
Move braces to their own lines.

Make sure to check our coding conventions: https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines

Reply:
Sorry about that, I've read it now.
I'll fix it.